src/ontodia/viewUtils/connectionsMenu.tsx

```diff
@@ -325,18 +325,20 @@ class ConnectionsMenuMarkup extends React.Component<ConnectionsMenuMarkupProps,
     };
 
     private getBreadCrumbs = () => {
-        return (this.props.objectsData && this.state.panel === 'objects' ?
-            <span className='ontodia-connections-menu_bread-crumbs'>
-                <a className='ontodia-connections-menu__link' onClick={this.onCollapseLink}>Connections</a>{'\u00A0' + '/' + '\u00A0'}
-                {
-                    chooseLocalizedText(
-                        this.props.objectsData.linkDataChunk.link.label,
-                        this.props.lang
-                    ).text.toLowerCase() + ` (${this.props.objectsData.linkDataChunk.direction})`
-                }
-            </span>
-            : ''
-        );
+        if (this.props.objectsData && this.state.panel === 'objects') {
+            const link = this.props.objectsData.linkDataChunk.link;
+            const localizedText = (link.label.length > 0 ?
+                chooseLocalizedText(link.label, this.props.lang).text : uri2name(link.id)
+            ).toLowerCase();
+
+            return <span className='ontodia-connections-menu_bread-crumbs'>
+                <a className='ontodia-connections-menu__link' onClick={this.onCollapseLink}>Connections</a>
+                {'\u00A0' + '/' + '\u00A0'}
+                {`${localizedText} (${this.props.objectsData.linkDataChunk.direction})`}
+            </span>;
+        } else {
+            return undefined;
```

Comment:
Use `null` instead of `undefined` when returning a React component (to be consistent with `render()` method: it's disallowed to return `undefined` from it).